files/en-us/web/css/css_lists_and_counters/consistent_list_indentation/index.html

```diff
@@ -44,15 +44,15 @@ <h2 id="Indenting_It_Twice">Indenting It Twice</h2>
 
 <p>If we dropped this list into a document as-is, there would be no apparent indentation and the markers would be in danger of falling off the left edge of the browser window.</p>
 
-<p>In order to avoid this and get some indentation, there are really only three options available to browser implementors.</p>
+<p> To avoid this and get some indentation, there are only three options available to browser implementors.</p>
 
 <ol>
  <li>Give each <code>&lt;li&gt;</code> element a left margin.</li>
  <li>Give the <code>&lt;ul&gt;</code> element a left margin.</li>
  <li>Give the <code>&lt;ul&gt;</code> element some left padding.</li>
 </ol>
 
-<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer for Windows and Macintosh, and Opera. The third was adopted by Gecko, and by extension all the browsers that embed it.</p>
+<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer for Windows and Macintosh, and Opera. The third was adopted by Gecko and by extension all the browsers that embed it.</p>
```

Comment:
```suggestion
<p>As it turns out, nobody seems to have used the first option. The second option was taken by Internet Explorer, and Opera. The third was adopted by Firefox.</p>
```
Embedded Geckos and IE for Mac are long dead. Let's simplify a bit.

Reply:
Oh, I don't know about thanks.